Nushell does not compile with `table-pager` feature

**Describe the bug**
Nushell fails to compile with the `table-pager` feature enabled.

**To Reproduce**
Steps to reproduce the behavior:
1. `cargo build --features=table-pager`

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
Error message:
```
error[E0053]: method `handle_input` has an incompatible type for trait
   --> crates/nu-command/src/commands/viewers/table/command.rs:314:17
    |
314 |             ev: Event,
    |                 ^^^^^
    |                 |
    |                 expected enum `crossterm::event::Event`, found a different enum `crossterm::event::Event`
    |                 help: change the parameter type to match the trait: `crossterm::event::Event`
    |
    = note: expected fn pointer `fn(&MinusInputHandler, crossterm::event::Event, _, SearchMode, LineNumbers, _) -> std::option::Option<_>`
               found fn pointer `fn(&MinusInputHandler, crossterm::event::Event, _, SearchMode, LineNumbers, _) -> std::option::Option<_>`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0053`.
error: could not compile `nu-command`
```

**Configuration (please complete the following information):**

Latest main:

```
> version | pivot
  #         Column0                                                         Column1
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   0   version              0.33.1
   1   branch               main
   2   short_commit         f603b7ef
   3   commit_hash          f603b7ef8b320f8c8e9f8bed54e79b0adb1c3ffb
   4   commit_date          2021-07-05 22:21:28
   5   build_os             linux-x86_64
   6   rust_version         rustc 1.53.0 (53cb7b09b 2021-06-17)
   7   rust_channel         stable (default)
   8   cargo_version        cargo 1.53.0 (4369396ce 2021-04-27)
   9   pkg_version          0.33.1
  10   build_time           2021-07-06 10:26:45
  11   build_rust_channel   release
  12   features             clipboard-cli, ctrlc, default, ptree, rustyline, term, trash, uuid, which, zip
  13   installed_plugins    binaryview, chart bar, chart line, fetch, from bson, from sqlite, inc, match, post, ps, query json, s3,
                            selector, start, sys, textview, to bson, to sqlite, tree, xpath
```

--- a/crates/nu-command/Cargo.toml
+++ b/crates/nu-command/Cargo.toml
@@ -37,7 +37,7 @@ calamine = "0.18.0"
 chrono = { version="0.4.19", features=["serde"] }
 chrono-tz = "0.5.3"
 codespan-reporting = "0.11.0"
-crossterm = { version="0.20.0", optional=true }
+crossterm = { version="0.19.0", optional=true }
 csv = "1.1.3"
 ctrlc = { version="3.1.7", optional=true }
 derive-new = "0.5.8"

